### ISSUE TITLE
Build error with dpkg-buildpackage

### ISSUE THREAD

**Issue by panzhitian** [shadowsocks/shadowsocks-libev#715]
```
Making check in src
make[2]: Entering directory `/root/shadowsocks-libev/src'
make[2]: Nothing to be done for `check'.
make[2]: Leaving directory `/root/shadowsocks-libev/src'
Making check in doc
make[2]: Entering directory `/root/shadowsocks-libev/doc'
make[2]: *** No rule to make target `check'.  Stop.
make[2]: Leaving directory `/root/shadowsocks-libev/doc'
make[1]: *** [check-recursive] Error 1
make[1]: Leaving directory `/root/shadowsocks-libev'
dh_auto_test: make -j1 check returned exit code 2
make: *** [build] Error 29
dpkg-buildpackage: error: debian/rules build gave error exit status 2
```
and it seems `dh_auto_clean` returns errors in doc folder as well.

**Comment by hosiet** [comment]
I am not sure about the reason of calling `make check`. It seems that
normal building process will not call this target.

Would you mind providing more information, such as the output of `cat
/etc/issue`, the `dpkg-buildpackage` command used to build the package and
its verbose output, etc.

On the other hand, it is a real problem since the Makefile in /doc does not
contain any check target. I am wondering if we could replace handwritten
Makefile.in with automake.

在 2016年7月10日週日 09:54，PoonChiTim <notifications@github.com> 寫道：

**Comment by panzhitian** [comment]
#### dirty build
http://pastebin.com/raw/wu37D3xT
```
 dpkg-source -i --before-build shadowsocks-libev
 debian/rules clean
make[2]: *** No rule to make target `distclean'.  Stop.
make[1]: *** [distclean-recursive] Error 1
dh_auto_clean: make -j1 distclean returned exit code 2
make: *** [clean] Error 29
dpkg-buildpackage: error: debian/rules clean gave error exit status 2
```
#### clean build
http://pastebin.com/raw/55adH3c9
```
make[2]: *** No rule to make target `check'.  Stop.
make[1]: *** [check-recursive] Error 1
dh_auto_test: make -j1 check returned exit code 2
make: *** [build] Error 29
dpkg-buildpackage: error: debian/rules build gave error exit status 2
```

**Comment by hosiet** [comment]
You haven't pasted your distribution version and build command.

在 2016年7月10日週日 10:42，PoonChiTim <notifications@github.com> 寫道：

**Comment by panzhitian** [comment]
I build on `Debian 7` with command `dpkg-buildpackage -b -us -uc -i`.
It has nothing to do with the build environment.
The errors appear on the deb packing process when the `dh_auto_test` calls `check`
and `dpkg-buildpackage` will call `clean` before it build a new pkg.

**Comment by madeye** [comment]
Fixed via 6923a8909b3c4cb5a596ad813ad538cc542e2a4e
